README.md

```diff
@@ -1,35 +1,39 @@
 ### About the dataset
-LibriMix is an open source dataset for source separation. 
-It's derived from LibriSpeech clean and WHAM noise. It's meant
-to offer an alternative or at least a complement to WSJ0 dataset.
+LibriMix is an open source dataset for source separation in noisy 
+environments. It is derived from LibriSpeech signals (clean subset) 
+and WHAM noise. It offers a free alternative to the WHAM dataset 
+and complements it. It will also enable cross-dataset experiments.
+
+### Generating LibriMix
+To generate LibriMix, clone the repo and run the main script : 
+[`generate_librimix.sh`](./generate_librimix.sh)
+
+```
+git clone https://github.com/JorisCos/LibriMix
+cd LibriMix 
+./generate_librimix.sh storage_dir n_src
```

Comment:
It's easier in this way and the repo itself doesn't weight much so that's fine to clone it locally

Reply:
Yeah you are right.